src/styles/blocks/_main.scss

```diff
@@ -0,0 +1,52 @@
+@import "variables";
+@import "mixins";
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 10px;
+  width: $side-size * 7 + $gap-for-day * 7;
+  position: relative;
+  left: 50%;
+  transform: translate(-50%,50%);
+  gap: $gap-for-day;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include square($side-size);
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    transition: 0.5s;
```

Comment:
you should specify transition-property(ies) always

Reply:
ok! I'll do this and in my next works!